testing/src/cypress/custom-command/institution/common-helper-functions.cy.ts

```diff
@@ -12,15 +16,17 @@ export default class InstitutionHelperActions {
     ];
   }
   getBaseUrlForTestEnv() {
-    return Cypress.env("TEST").BASE_URL;
+    return ApiData.testEnv.baseUrl;
   }
 
   getLoginUrlForTestEnv() {
-    return `${this.getBaseUrlForTestEnv()}/institution/login`;
+    return `${this.getBaseUrlForTestEnv()}${
+      ApiData.endPoints.institutionLogin
+    }`;
   }
 
   getApiUrlForTest() {
-    return `${this.getBaseUrlForTestEnv()}/api/institutions`;
+    return `${this.getBaseUrlForTestEnv()}${ApiData.endPoints.institutionsApi}`;
   }
 
   getUserNameSingleLocation() {
```

Comment:
Do we still need the getUserName() and getPassword() in the common helper as these proerties will be used only inside the getToken method?

Reply:
i think we still need them as to get different user details like institution user vs student user etc